test/io-tests/fixtures.yaml

```diff
@@ -173,5 +173,4 @@ invalidroleclaimkeys:
   - '.role##'
   - '.my_role;;domain'
   - '.#$$%&$%/'
-  - ''
```

Comment:
empty string is treated as Nothing now, so the default `.role` applies. This makes it easier to reuse the `optString` function. I think it's fine.